src/Integration.UnitTests/Binding/BindingWorkflowTests.cs

```diff
@@ -278,23 +282,181 @@ public void BindingWorkflow_SilentSaveSolutionIfDirty()
             // Verify
             this.outputWindowPane.AssertOutputStrings(0);
         }
+
+        [TestMethod]
+        public void BindingWorkflow_GetBindingLangauges_ReturnsDistinctLanguagesForProjects()
+        {
+            // Setup
+            var testSubject = this.CreateTestSubject();
+
+            var csProject1 = new ProjectMock("cs1.csproj");
+            var csProject2 = new ProjectMock("cs2.csproj");
+            var csProject3 = new ProjectMock("cs3.csproj");
+            csProject1.SetCSProjectKind();
+            csProject2.SetCSProjectKind();
+            csProject3.SetCSProjectKind();
+            var vbNetProject1 = new ProjectMock("vb1.vbproj");
+            var vbNetProject2 = new ProjectMock("vb2.vbproj");
+            vbNetProject1.SetVBProjectKind();
+            vbNetProject2.SetVBProjectKind();
+            var projects = new[]
+            {
+                csProject1,
+                csProject2,
+                vbNetProject1,
+                csProject3,
+                vbNetProject2
+            };
+            testSubject.BindingProjects.AddRange(projects);
+
+            var expectedLanguages = new[] { Language.CSharp, Language.VBNET };
+
+            // Act
+            var actualLanguages = testSubject.GetBindingLanguages();
+
+            // Verify
+            CollectionAssert.AreEqual(expectedLanguages, actualLanguages.ToArray(), "Unexpected lanuages for binding projects");
```

Comment:
AreEquivalent might be more robust and appropriate unless we care about order (?)